extensions/wikia/AdEngine/package.json

```diff
@@ -19,7 +19,8 @@
   },
   "scripts": {
     "build": "webpack -p",
-    "watch": "webpack -p -w"
+    "watch": "webpack -p -w",
+    "postinstall": "npm dedupe; npm prune"
```

Comment:
Automaticaly dedupe dependencies. Unnecessary packages in nested node_modules causes redundancy in a target build because of resolving hierarchy (same thing happens if packages are linked). We can use webpack aliases, but it isn't a nice sulution.